presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -61,4 +62,16 @@ public BaseJdbcConfig setConnectionPassword(String connectionPassword)
         this.connectionPassword = connectionPassword;
         return this;
     }
+
+    public boolean isMapLowercaseTableNames()
+    {
+        return mapLowercaseTableNames;
+    }
+
+    @Config("connection-map-tables")
+    public BaseJdbcConfig setMapLowercaseTableNames(boolean mapLowercaseTableNames)
```

Comment:
My thought for having a flag is to make this an optional feature for people, so as to preserve current behavior for those who don't want to use this. I'm very happy to make it default on, or even just always on without a flag.